.github/workflows/image-publish.yaml

```diff
@@ -5,8 +5,7 @@ on:
     types: [published, edited]
 
 env:
-  dockerhub_namespace: kloeckneri
-  manufacturer: kloeckner-i
+  manufacturer: db-operator
```

Comment:
this stays?

Reply:
Nope, I'll fix

Reply:
Hm, I've read it wrong. I'd leave it like this, because I want to update all the actions for all projects and I think it should be done later. 
What exactly don't you like here?